src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount = [20, 50];
+  const dayCost = 40;
+  let totalCost = 0;
+
+  if (days < 3) {
+    totalCost = dayCost * days;
+  } else if (days >= 3 && days < 7) {
+    totalCost = (dayCost * days) - discount[0];
+  } else {
+    totalCost = (dayCost * days) - discount[1];
+  }
```

Comment:
please pay attention  to this previous comment 🙂
<img width="801" alt="Screenshot 2023-03-05 at 12 37 37" src="https://user-images.githubusercontent.com/32853034/222958207-ce6d04ac-483e-437b-b23a-0ae4142e576b.png">

you can return value right in `if` block and get rid of extra `else` block